components/SlateRenderer/SlateRenderer.module.scss

```diff
@@ -11,8 +11,10 @@
         .prezly-slate-document {
             color: inherit;
 
-            > *:last-child {
-                margin-bottom: 0 !important;
+            @include tablet-up {
```

Comment:
Why was this added?

If there's not enough space on the bottom, it should be changed with padding in the Layout or Story component

Reply:
It's because I've noticed it looks bad on mobile:

![Screenshot 2023-05-24 at 9 31 32](https://github.com/prezly/theme-nextjs-lena/assets/4209081/82205116-32f5-4f32-8975-326a9659650b)

On desktop, there's enough space at the bottom due to padding but on mobile, [that padding is reset to 0](https://github.com/prezly/theme-nextjs-lena/blob/main/modules/Layout/Layout.module.scss#L21).

So I guess a better solution would be to keep the bottom padding? Something like 2rem looks like this:
![Screenshot 2023-05-24 at 9 33 54](https://github.com/prezly/theme-nextjs-lena/assets/4209081/1f56443e-8474-47ef-98f0-8961d5891fd2)

Reply:
Yeah, that works well! No idea why the padding was reset for mobile 🤷🏻